### PR TITLE
 Added code of daily Coding Exercise-12

### PR DESCRIPTION
**Pull request by anushreeSonak**

### CODE REVIEW

File: First/dailyExercises/ReverseLetters.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package firstPackage.First.dailyExercises;
+
+public class ReverseLetters {
+    public static void main(String[] args) {
+        String mockData = "Anu";
+        String[] arrayData = mockData.split(" ");
```
Comment (amitsahu8) [inline First/dailyExercises/ReverseLetters.java]: no need of split

File: First/dailyExercises/ReverseLetters.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        for (String temp : arrayData) {
+            System.out.println(temp);
+        }
+        for (int index = 0; index < 1; index++) {
+            char[] letter = arrayData[index].toCharArray();
+            for (int subIndex = letter.length - 1; subIndex >= 0; subIndex--) {
```
Comment (amitsahu8) [inline First/dailyExercises/ReverseLetters.java]: no need of nested for loop

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: First/dailyExercises/ReverseLetters.java (hunk 1 above)
Comment (anushreeSonak) [reply First/dailyExercises/ReverseLetters.java]: DONE !

File: First/dailyExercises/ReverseLetters.java (hunk 2 above)
Comment (anushreeSonak) [reply First/dailyExercises/ReverseLetters.java]: DONE !

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**